src/cljs/athens/views.cljs

```diff
@@ -88,9 +90,8 @@
       [alert]
       (if @loading
         [:div
-          [style/loading-css]
-          [:h4 {:id "loading-text"} "Loading... (at least it'll be faster than Roam)"]
-        ]
+         [style/loading-css]
+         [:h4 {:id "loading-text"} "Loading database... (at least it'll be faster than Roam)"]]
```

Comment:
Now this text is duplicated? Do we need it in index.html?

Reply:
Yeah, also can we remove the Roam reference in all cases? I was feeling cheeky that day. 😂